src/components/ScrollableContainer.tsx

```diff
@@ -5,13 +5,15 @@ import { getAbsoluteHeightBelow, getParentsBottomSpacing } from "util/helpers";
 interface Props {
   children: ReactNode;
   dependencies: DependencyList;
-  belowId?: string;
+  belowIds?: string[];
+  viewPortHeightLimit?: number;
```

Comment:
This is a bit of a wild concept to introduce here. I think this was added to `EditGroupPermissionsPanel`, but maybe we can just not use the `ScrollableTable` in it? Then we can remove the `viewPortHeightLimit` from this as well as from the `ScrollableTable` component.